fairlearn/postprocessing/_plotting.py

```diff
@@ -104,3 +108,269 @@ def plot_threshold_optimizer(threshold_optimizer, ax=None, show_plot=True):
 
     if show_plot:
         plt.show()
+
+
+"""Utilities to plot information about the output of an estimator"""
+
+
+def _check_A(A):
```

Comment:
The whole `A` terminology is based on some papers and I don't think we should propagate that any further than we've already done. We try to use `sensitive_features` all throughout the code. If you see anything suggesting otherwise feel free to file an issue or fix it yourself with a PR.

The reason is just that "A" isn't terribly obvious. I can see why researchers choose it in papers, but in code you don't always have the luxury of explaining what every letter means. It's not as pervasive as X and y, either.